pom.xml

```diff
@@ -51,18 +52,49 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct</artifactId>
+            <version>${org.mapstruct.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>${liquibase.version}</version>
```

Comment:
You can alias ${liquibase.version} of version when you added the alias in properties pom.file. 
I didn't find it. Also like Nazar said you don't need to mentioned here the version.